glutin/src/api/egl/sync.rs

```diff
@@ -0,0 +1,166 @@
+//! EGL Sync Fences.
+
+use std::ffi::c_void;
+use std::mem::MaybeUninit;
+use std::sync::Arc;
+use std::time::Duration;
+
+use glutin_egl_sys::egl::types::{EGLenum, EGLint};
+
+use super::display::DisplayInner;
+use super::{egl, ErrorKind, VERSION_1_5};
+use crate::error::Result;
+
+/// EGL sync object.
+#[derive(Debug, Clone)]
+pub struct Sync(pub(super) Arc<Inner>);
+
+impl Sync {
+    /// Insert this sync into the currently bound context.
```

Comment:
The wording is a bit tricky for this one, because context could assume glutin's context.

```suggestion
    /// Insert this sync into the currently bound *server* context.
```